src/main/scala/io/iohk/ethereum/jsonrpc/EthJsonMethodsImplicits.scala

```diff
@@ -401,4 +401,17 @@ object EthJsonMethodsImplicits extends JsonMethodsImplicits {
     def encodeJson(t: GetTransactionCountResponse): JValue = encodeAsHex(t.value)
   }
 
+  implicit val eth_sign = new JsonDecoder[SignRequest] {
+    override def decodeJson(params: Option[JArray]): Either[JsonRpcError, SignRequest] =
+      params match {
+        case Some(JArray(JString(message) :: JString(addr) :: _)) =>
```

Comment:
Maybe the order of the parameters should be `[addr, message]`? It seems to be that way in https://github.com/ethereum/wiki/wiki/JSON-RPC#eth_sign.